doc/advanced_configuration.rst

```diff
@@ -397,6 +398,50 @@ The default behavior is ``sphinx_gallery_thumbnail_number = 1``. See
 :ref:`sphx_glr_auto_examples_plot_choose_thumbnail.py` for an example
 of this functionality.
 
+.. _binder_links:
+
+Generate Binder links for gallery notebooks
```

Comment:
A mentioned before and in consistence with the warning. Can we put (Experimental) at the end of the title?